### ISSUE TITLE
Example relative imports

### ISSUE THREAD

**Issue by timcash** [livekit/client-sdk-js#254]
in the example it uses
```
} from '../src/index';
import { LogLevel } from '../src/logger';
import { TrackSource } from '../src/proto/livekit_models';
```
I think it would be more friendly as a sample if it imported everything from the library no? Otherwise we need to copy paste those files to use them?

**Comment by davidzhao** [comment]
I haven't tried.. but would importing from the library require it to be built or installed?

If it could preserve the same functionality (i.e. user clones repo, runs yarn sample and sees it), I would be supportive!

**Comment by lukasIO** [comment]
Hi @timcash,

as @davidzhao pointed out, we currently trigger rebuilds of the sample based on changes in the source code and we'd like to keep it that way (the way it is set up, those dev builds are a lot faster than the production library build).

To address your immediate issue, I opened https://github.com/livekit/client-sdk-js/pull/255 to at least reduce everything to one import location. That way you can simply replace `../src/index` with `livekit-client` in your application.

**Comment by lukasIO** [comment]
closed via #255
